locales/en/app.ftl

```diff
@@ -33,6 +33,7 @@ oauth-invalid-session = Invalid session
 scan-title = {-product-name} : Scan Results
 
 user-add-invalid-email = Invalid Email
+user-add-too-many-emails = You are monitoring the maximum number of email addresses.
```

Comment:
I assume you'd get this after trying to add another email, so maybe "you are already…"?

Reply:
+1

Reply:
The message could potentially show up in 2 places.

1. It always replaces the "add another email form" when the user already has the maximum number
2. If the user somehow hacks together a form request to add another email address when they're at the maximum, they will get an error page displaying this same message.

So, since the 2nd case should never really happen, I dropped the "already" term.